lib/badge-data.js

```diff
@@ -0,0 +1,99 @@
+const logos = require('./load-logos')();
+
+function toArray (val) {
+  if (val === undefined) {
+    return [];
+  } else if (Object(val) instanceof Array) {
+    return val;
+  } else {
+    return [val];
+  }
+}
+
+function isDataUri(s) {
+  return s !== undefined && /^(data:)([^;]+);([^,]+),(.+)$/.test(s);
+}
+
+function hasPrefix (s, prefix) {
```

Comment:
For consistency's sake, could you avoid having a space between the function name and the opening parenthesis?
eg. `function hasPrefix(s, prefix)`.

Reply:
Sure, will change them. I don't think debating is worth it. #1082 is a more productive way forward.